clean-abap/CleanABAP.md

```diff
@@ -4187,21 +4187,19 @@ DATA reader TYPE REF TO /clean/reader.
 > [Clean ABAP](#clean-abap) > [Content](#content) > [Formatting](#formatting) > [This section](#dont-chain-assignments)
 
 ```abap
-var2 = var3.
-var1 = var3.
-```
-
-```abap
-var1 = xsdbool( var2 = var3 ).
+" anti-pattern
+var1 = var2 = var3.
 ```
 
 Chained assignments usually confuse the reader. Besides, the inline declaration doesn't work in any position of a multiple assignment.
 
 ```abap
-" anti-pattern
-var1 = var2 = var3.
+var2 = var3.
+var1 = var3.
 ```
 
+Furthermore, the anti-pattern looks ambiguous because `=` is used for comparisons and assignments in ABAP. It looks similar to how other programming languages implement comparisons, for example `a = ( b == c )` in JavaScript. 
+
```

Comment:
The ABAP equivalent would be `a = xsdbool( b = c ).`

Reply:
Do we really need this explanation though? I'd like to avoid repeating chapter "Use XSDBOOL to set Boolean variables"

Reply:
I thought it's useful to highlight the distinction between `a = b = c` as a chain and a boolean. If the intention is `a` is true if `b` equals `c`, then even `a = ( b = c ).` is not valid ABAP, therefore show what to do if that is the intent. But it was just a minor suggestions, I'm not too fussed either way.

Reply:
I've included the example along with a link to the corresponding chapter. OK to resolve now? :)

Reply:
The link is a great idea 🙂

Reply:
I was able to resolve this thread myself.